test/functional/test_runner.py

```diff
@@ -93,7 +93,6 @@
     'rpc_txoutproof.py',
```

Comment:
9e3770e51522083e150c53b29a1b1cf266d6e8d4 missing changes in init.cpp and test/functional/feature_asmap.py

Reply:
`init.cpp` fixes implemented in https://github.com/dashpay/dash/commit/36c368099a64a1dd888a1207600119e25adba856#diff-b1e19192258d83199d8adaa5ac31f067af98f63554bfdd679bd8e8073815e69dR1933-R1957. Commit misattribution due to rebase mixup